### PR TITLE
Feature/form

### PR DESCRIPTION
**Pull request by rrrainville**

### CODE REVIEW

File: .env
Comment (marceloglacial) [inline .env:1]: This is a sensitive information. 
Please delete this file and add it to gitignore

File: components/Form/Form.jsx [hunk 1]
```diff
@@ -192,51 +227,45 @@ const Form = (props) => {
...
+                            placeholder={getField(fields, 'linkedin').placeholder}
+                            value={formValues.linkedin_profile}
                             onChange={handleChange}
                         />
 
                         <FormInput
                             name='email'
-                            mandatory='yes'
+                            type='email'
+                            required='required'
```
Comment (marceloglacial) [inline components/Form/Form.jsx:241]: It's not working. We can still send the form without fill this field.

File: components/Form/Form.jsx [hunk 2]
```diff
@@ -34,95 +46,111 @@ const Form = (props) => {
 
     const handleSubmit = async (event) => {
```
Comment (marceloglacial) [inline components/Form/Form.jsx:47]: Add a "Sending" message on the button and disable it while is sending formData

**Review by marceloglacial — CHANGES_REQUESTED** [review]

File: components/Form/Form.jsx (hunk 1 above)
Comment (rrrainville) [reply components/Form/Form.jsx:241]: ontem estava funcionando... o q nao funciona?

Comment (rrrainville) [reply components/Form/Form.jsx:241]: acabei de rodar aqui e esta validando

File: components/Form/Form.jsx [hunk 3]
```diff
@@ -134,25 +162,28 @@ const Form = (props) => {
...
+                                    .map((province) => (
+                                        <option key={province} value={province}>
+                                            {province}
+                                        </option>
+                                    ))}
                             </select>
                         </div>
                         <FormInput
                             disabled={!formValues.residein_canada}
+                            required='required'
```
[the remark this answers is not in the document]
Comment (rrrainville) [reply components/Form/Form.jsx:182]: esta funcionando como esperado. o campo esta desabilitado, portanto, a validacao nao ocorre.

**Review by marceloglacial — APPROVED** [review]

**Pull request merged**